src/Navbar.js

```diff
@@ -1,91 +1,106 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import cx from 'classnames';
-import Col from './Col';
 import Icon from './Icon';
 
 class Navbar extends Component {
   constructor(props) {
     super(props);
-    this.renderSideNav = this.renderSideNav.bind(this);
   }
 
   componentDidMount() {
-    if (typeof $ !== 'undefined') {
-      $('.button-collapse').sideNav(this.props.options);
+    const { options } = this.props;
```

Comment:
could we set this default to `{}` just to not init with undefined?

Reply:
Did that in `Navbar.defaultProps`